pixel-saver@deadalnix.me/decoration.js

```diff
@@ -122,6 +129,10 @@ function getOriginalState(win) {
 	if (win._pixelSaverOriginalState !== undefined) {
 		return win._pixelSaverOriginalState;
 	}
+
+	if (!win.decorated) {
```

Comment:
Alright. Sounds good. I'll test this on my machine when I finally get home but it looks good to merge. Thanks.